### DISCUSSION TITLE
How to enter date in js and using supabase timestamp time?

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#2839] [Questions] [answered]
** I tried using "Date.now()" and it render error

** Error
{hint: "Perhaps you need a different "datestyle" setting.", details: null, code: "22008",…}
code: "22008"
details: null
hint: "Perhaps you need a different \"datestyle\" setting."
message: "date/time field value out of range: \"1628948903422\""

**Comment by soedirgo** [comment, marked as answer]
Timestamps have to be in ISO string format, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString.

**Reply by donPuerto** [reply]
Thanks it works
```js
{
'updated_at': ((new Date()).toISOString()).toLocaleString('zh-TW')
}
